Cargo.toml

```diff
@@ -1,9 +1,12 @@
 [package]
 name = "xenstore-rs"
-version = "0.7.0"
-authors = ["Mathieu Tarral <mathieu.tarral@protonmail.com>"]
+version = "1.0.0-dev"
```

Comment:
Why mark this update as 1.0.0 release ?

Reply:
I felt that it would be a good candidate for 1.0 assuming no changes to the library's API. But I am ok too with 0.8 for having future iterations on the design.

Reply:
Yeah let's go with 0.8